.github/workflows/release.yml

```diff
@@ -0,0 +1,67 @@
+name: Release
+
+on:
+  release:
```

Comment:
Lately I've been preferring the workflow_dispatch event — it allows me to type in the version number I want and both PyPI and GH releases get created with that.
I can also make Git updates like generating the changelog, bumping the version, making a tag, and pushing that as a part of the workflow, making sure all the steps are automated, lowering the possibility of human errors as much as possible.